.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Backend workflow for Golang using Snyk
```

Comment:
You can leave the name short, like **Vulnerability Scan**

Reply:
okay